.github/workflows/docs.yml

```diff
@@ -37,21 +37,22 @@ jobs:
       with:
         hugo-version: '0.134.2'
 
-    - name: Build theme
+    - name: Get theme
       working-directory: ./docs/themes/geekdoc
-      run: |
-        npm install
-        npm run build
+      run: curl -L https://github.com/thegeeklab/hugo-geekdoc/releases/latest/download/hugo-geekdoc.tar.gz | tar -xz --strip-components=1
 
     - name: Check out GitHub page repo
       uses: actions/checkout@v4
+      if: github.ref == 'refs/heads/master'
       with:
         repository: emacs-eask/emacs-eask.github.io
         path: emacs-eask.github.io
         token: ${{ secrets.PAT }}
 
     - name: Build doc artifacts with Hugo
-      run: hugo --destination ../emacs-eask.github.io --minify
+      run: |
+        mkdir ../emacs-eask.github.io
```

Comment:
In case the previous step did not create the dir.

Reply:
It will be created with the hugo command, but it's all good. ;)